assets/src/components/parts/janus-Carousel/Carousel.tsx

```diff
@@ -56,6 +63,32 @@ const Carousel: React.FC<any> = (props) => {
       : `calc(${height} - ${MAGIC_NUMBER}px)`,
   };
 
+  useEffect(() => {
+    props.onReady({
+      activityId: `${id}`,
+      partResponses: [
```

Comment:
in my latest branch (not yet master) I've been doing it like this:
https://github.com/Simon-Initiative/oli-torus/blob/feature/adaptive-activity-3/assets/src/components/parts/janus-input-text/InputText.tsx#L36